frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -0,0 +1,52 @@
+export const listPage = {
+  titleShouldBe: (title: string) =>
+    cy.get(`[data-test="resource-title"]`).should('have.text', title),
+  projectDropdownShouldExist: () => cy.byLegacyTestID('namespace-bar-dropdown').should('exist'),
+  projectDropdownShouldContain: (name: string) =>
+    cy.byLegacyTestID('namespace-bar-dropdown').contains(name),
+  projectDropdownShouldNotExist: () =>
+    cy.byLegacyTestID('namespace-bar-dropdown').should('not.exist'),
+  clickCreateYAMLButton: () => {
+    // eslint-disable-next-line promise/catch-or-return
+    cy.contains('button', 'Create')
+      .click()
+      .get('body')
+      .then(($body) => {
```

Comment:
We should consider just using async/await instead of promises.

Reply:
There have been a lot of requests for this in the Cypress chats.   Pls see ["Commands Are Not Promises"](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Commands-Are-Not-Promises) and also this [more specific discussion](https://github.com/cypress-io/cypress/issues/1417#issuecomment-370860080). -thanks